src/main/java/seedu/manager/Main.java

```diff
@@ -29,8 +29,8 @@ private static void runCommandLoop() {
         boolean isGettingCommands = true;
         while (isGettingCommands){
             String userCommandText = ui.getCommand();
-            command = new Parser().parseCommand(userCommandText);
-            command.setData(events);
+            command = new Parser().parseCommand(userCommandText, events);
+            //command.setData(events);
```

Comment:
I think it would be better if the event list was passed in using the existing setData() method, rather than through the constructor?